sync/firewall/firewall.go

```diff
@@ -26,13 +27,25 @@ type Firewall struct {
 	transactionRateLimit *ratelimit.RateLimit
 	consensusRateLimit   *ratelimit.RateLimit
 	state                state.Facade
+	ipBlocker            *ipblocker.IPBlocker
 	logger               *logger.SubLogger
 }
 
 func NewFirewall(conf *Config, net network.Network, peerSet *peerset.PeerSet, st state.Facade,
 	log *logger.SubLogger,
-) *Firewall {
-	conf.LoadDefaultBlackListAddresses()
+) (*Firewall, error) {
+	if err := conf.LoadDefaultBlackListAddresses(); err != nil {
+		return nil, err
+	}
+
+	if err := conf.BasicCheck(); err != nil {
```

Comment:
We have this check once we run the node, look [here](https://github.com/pactus-project/pactus/blob/9c5d7bd4c379207ee6378149dcedbb5adeb0e90b/cmd/cmd.go#L385)
Please add this code inside here: https://github.com/pactus-project/pactus/blob/e947cb662e737233d2c91664f0558fb75d4454b4/sync/config.go#L43